tensorflow_federated/python/learning/keras_utils.py

```diff
@@ -230,7 +277,7 @@ def report(accumulators):
 class _KerasModel(model_lib.Model):
   """Internal wrapper class for tf.keras.Model objects."""
 
-  def __init__(self, inner_model, dummy_batch, loss_fn, metrics):
+  def __init__(self, inner_model, dummy_batch, loss_fns, loss_weights, metrics):
```

Comment:
```suggestion
  def __init__(self, inner_model, dummy_batch, loss_fns, loss_weights=None, metrics=None):
```

Lets add defaults for everything after `loss_fns`